### DISCUSSION TITLE
Add: Nickelodeon HD US East & Cartoon Network US East, Teennick USA

### DISCUSSION THREAD

**Discussion by Westh977** [iptv-org/community#1129] [Q&A]
Hello, I Am Looking for Nickelodeon US HD East & Cartoon  Network US, Teennick HD US East And Then It's Missing Can You Guys Find It Also Can You Add PBS Retro Please Thank You

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
